### PR TITLE
Migrate to Maven Central Repository

### PR DESCRIPTION
**Pull request by ershkia**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -1,10 +1,15 @@
-name: Deploy to BinTray
+name: Deploy to MavenCentral
+
+#on:
+#  release:
+#    types:
+#      - published
 
 on:
```
Comment (ershkia) [inline .github/workflows/deploy.yml:8]: revert this before merge

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -13,7 +18,7 @@ jobs:
     
     - uses: actions/checkout@master
       with:
-        ref: 'master'
+        ref: 'migration-to-jfrog'
```
Comment (ershkia) [inline .github/workflows/deploy.yml:21]: revert this before merge

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -28,14 +33,12 @@ jobs:
     - name: Sync Submodules
       run: git submodule sync --recursive && git submodule update --recursive --init
           
-    - name: Push to BinTray
+    - name: Publish Package
       working-directory: ./MobileBuy
+      run: ./gradlew uploadArchives
```
[the remark this answers is not in the document]
Comment (ershkia) [reply .github/workflows/deploy.yml:38]: yes because the other module is the sample app that I believe we don't release.

**Comment by jaredh** [comment]
I'm opening a new PR and cleaning up the git history for this branch. I probably shouldn't be able to review my own changes as well. 😇

**Pull request closed without merging**